patches/minecraft/net/minecraft/item/crafting/FurnaceRecipes.java.patch

```diff
@@ -1,20 +1,11 @@
 --- ../src-base/minecraft/net/minecraft/item/crafting/FurnaceRecipes.java
 +++ ../src-work/minecraft/net/minecraft/item/crafting/FurnaceRecipes.java
-@@ -113,6 +113,7 @@
+@@ -20,7 +20,7 @@
  
-     public void func_151394_a(ItemStack p_151394_1_, ItemStack p_151394_2_, float p_151394_3_)
```

Comment:
Removing a vanilla function breaks binary compatibility

Reply:
No, that is not removal. It is that old methods displayed in patches are no longer displayed because that piece of patch is now away.

Reply:
Ah, right. Patches are a pain to read